pkg/controller/plan/adapter/ovirt/builder.go

```diff
@@ -525,7 +525,12 @@ func (r *Builder) PersistentVolumeClaimWithSourceRef(da interface{}, storageName
 	diskAttachment := da.(model.XDiskAttachment)
 
 	// We add 10% overhead because of the fsOverhead in CDI, around 5% to ext4 and 5% for root partition.
-	diskSize := int64(float64(diskAttachment.Disk.ProvisionedSize) * 1.1)
+	diskSize := diskAttachment.Disk.ProvisionedSize
+	// Accounting for fsOverhead is only required for `volumeMode: Filesystem`
+	if *volumeMode == core.PersistentVolumeFilesystem {
```

Comment:
can volumeMode be nil here?

Reply:
based on `getStorageProfileModes` which we get the volumeMode from if it's nil there, we set it with filesystem.

Reply:
@liranr23 thanks for checking